packages/backend/src/services/DashboardService/DashboardService.ts

```diff
@@ -83,12 +85,17 @@ export class DashboardService {
         spaceUuid: string,
         userUuid: string,
     ): Promise<boolean> {
+        let space: Space;
+
         try {
-            const space = await this.spaceModel.getFullSpace(spaceUuid);
-            return hasSpaceAccess(space, userUuid);
+            space = await this.spaceModel.getFullSpace(spaceUuid);
         } catch (e) {
+            Sentry.captureException(e);
+            console.error(e);
             return false;
         }
+
+        return hasSpaceAccess(space, userUuid);
```

Comment:
is this not doing the opposite ?  excluding `hasSpaceAccess` from the try-catch. 

What's the error exactly ?

Reply:
@rephus Oliver and I had a discussion:

we are trying to figure out a problem with this user (you can have a look to our intercom)

- user created space (is the owner)
- space is shared and can't create dashboard inside
- getting error "You don't have access to the space this dashboard belongs to"

that happens if space does not exist with that UUID (we can be sure that space exists)
now what we are trying to do is to check if `hasSpaceAccess` is throwing an exception